cpp/src/Ice/Service.cpp

```diff
@@ -176,7 +176,7 @@ namespace
             // Don't need to use a wide string converter as the wide string is passed
             // to Windows API.
             //
-            LONG err = RegCreateKeyExW(
+            LSTATUS err = RegCreateKeyExW(
```

Comment:
The Windows API is weird: some error codes that can be "printed" with FormatMessage are not DWORDs.